clouds/azure/session_clients.py

```diff
@@ -0,0 +1,29 @@
+from azure.identity import ClientSecretCredential
+from azure.mgmt.subscription import SubscriptionClient
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.redhatopenshift import AzureRedHatOpenShiftClient
+
+
+def get_azure_credentials(tenant_id, client_id, client_secret):
+    return ClientSecretCredential(
+        tenant_id=tenant_id,
+        client_id=client_id,
+        client_secret=client_secret,
+    )
+
+
+def get_aro_client(credential, subscription_id):
```

Comment:
```suggestion
def aro_client(credential, subscription_id):
```

applicable for all functions here

Reply:
done